assets/js/just-the-docs.js

```diff
@@ -1,9 +1,9 @@
+---
+---
 // SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES.
 // All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
----
----
```

Comment:
This is the primary fix needed here. I think the inclusion of the copyright notice comments causes the Jekyll template engine not to trigger. You can see that currently, https://docs.rapids.ai/assets/js/just-the-docs.js appears to have raw template content. Moving the `---` up above the copyright notice comments appears to fix it.